js/pages/options.es6.js

```diff
@@ -14,6 +15,7 @@ function Options (ops) {
 
 Options.prototype = $.extend({},
     Parent.prototype,
+    mixins.setBrowserClassOnBodyTag,
```

Comment:
(I wouldn't add it to the Parent Page in base because this is far too specific to this particular application; its useful to think of the Parent Page - as in /js/base/page -- as a 3rd party library almost. Kind of like Backbone, Angular, Vue, etc. Only the really generic stuff goes there that governs app lifecycle goes there)